api/util/util.go

```diff
@@ -178,3 +178,36 @@ func CheckMaliciousRID(RID string, c echo.Context) error {
 	}
 	return nil
 }
+
+// CountRetireJSOccurrences remove duplicates and increment occurrences in each RetireJS vuln found
+func CountRetireJSOccurrences(retireJSresults []types.HuskyCIVulnerability) []types.HuskyCIVulnerability {
```

Comment:
The context of a util package is confusing (what is the relationship between the functions here?).
For the specific case of this function (`CountRetireJSOccurrences`), can this be moved to `api/analysis/retirejs.go` because it is related to the specific case of retireJS?

Reply:
It makes totally sense, @mdjunior! I have just made this refactoring here at aad620c.